posts/2024-06-21-june-meetup/index.qmd

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "SheffieldR : June 2024 Meetup"
+date:   2024-06-21 16:00:00 +0000
+categories: meetups
+image: image.png
+excerpt_separator: <!--more-->
+---
+
+This month, we're shaking things up with an exciting line-up of lightning talks presented by you, our vibrant community members! This is your chance to share your knowledge, showcase your projects, and discuss your favourite R topics in a quick, 5-10 minute format.
+
+If you have a fascinating R project you're working on, a useful package you can't live without, or a data analysis technique you think others should know about sign up to present a lightning talk and share it with the SheffieldR community!
```

Comment:
Could also add a link to the form here (have wrapped line at 120 as [`markdonwlint-cli2`](https://github.com/DavidAnson/markdownlint-cli2) is failing on pre-commit.ci here and a few other places, but these are minor and aren't a barrier to merging).

```suggestion
If you have a fascinating R project you're working on, a useful package you can't live without, or a data analysis
technique you think others should know about [sign up](https://forms.gle/YSWXarhwUpXZQp2J6) to present a lightning talk
and share it with the SheffieldR community!
```